edx_proctoring/templates/proctoring/proctoring_launch_callback.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+</body>
+<p>Your proctored exam has started. Please immediately go back to the website to enter into your exam</p>
```

Comment:
"Your proctoring session has started. Do not close this window. Return to the browser window where your course is running to take your exam."